### ISSUE TITLE
UDP server "ALL GROUP ON/OFF" are backwards

### ISSUE THREAD

**Issue by phigmeta** [sidoh/esp8266_milight_hub#65]
If you send the UDP signal for "ALL ON" (42 00 55 - All On) it turns everything off

If you Send the UDP signal for "ALL OFF" (41 00 55 - All Off) it turns everything on

Group UDP functions correctly:

45 00 55 - Zone 1 On
46 00 55 - Zone 1 Off
47 00 55 - Zone 2 On
48 00 55 - Zone 2 Off
49 00 55 - Zone 3 On
4A 00 55 - Zone 3 Off
4B 00 55 - Zone 4 On
4C 00 55 - Zone 4 Off

Version 5

Tested with app first, then with UDP Sender...

Used RGBW Bulbs with Zone 1 and RGBW LED Driver with Zone 2 

No further devices tested as I don't have any

**Comment by g-rom-info** [comment]
According to [limitless documentation](http://www.limitlessled.com/dev/), you are right !

> LimitlessLED RGBW
> -------------------
> 41 00 55 - All Off
> 42 00 55 - All On
> 43 00 55 - Speed Down (One Step Slower Disco)
> 44 00 55 - Speed Up (One Step Faster Disco)
> 45 00 55 - Zone 1 On
> 46 00 55 - Zone 1 Off
> .........

It can be easily corrected by editing vars on V5MiLightUdpServer.h

Replace
```
  UDP_RGBW_ALL_ON            = 0x41,
  UDP_RGBW_ALL_OFF           = 0x42,

```
By
```
  UDP_RGBW_ALL_ON            = 0x42,
  UDP_RGBW_ALL_OFF           = 0x41,
```

**Comment by sidoh** [comment]
Good find.

Happy to look at a PR if someone wants to toss one up. Otherwise I'll get to it over the next few days.

**Comment by phigmeta** [comment]
PR sent as requested, would have done that earlier... but for some reason, I fail at git.

**Comment by sidoh** [comment]
Thanks! Looks good. I'll test it with the bulbs I have later today.

**Comment by g-rom-info** [comment]
Just tested it with my bulbs and this quick fix, it's OK :)

**Comment by sidoh** [comment]
Gonna close this since it sounds like it's fixed. Thanks, @phigmeta!

**Comment by g-rom-info** [comment]
@sidoh 1.2.3 version still have this issue :-(

**Comment by sidoh** [comment]
Hmm... should've been fixed in 654b4f5380ae47ead5fe0d24cf7d0bba5a2b8e82. I'll test when I'm home. Maybe some later change swapped it back somehow.

**Comment by g-rom-info** [comment]
Yes but not in the master branch :) [See here](https://github.com/sidoh/esp8266_milight_hub/blob/master/lib/Udp/V5MiLightUdpServer.h).

**Comment by sidoh** [comment]
Argh. Crud. I must've done something goofy with git. I just manually reapplied it and released 1.2.4.
